service/dynamodb/expression/condition.go

```diff
@@ -12,12 +12,38 @@ type ConditionMode int
 const (
 	// UnsetCond will catch errors if users make an empty ConditionBuilder
 	UnsetCond ConditionMode = iota
```

Comment:
Would suggest not exporting these. As the end user is unable to use the values or type.